server/src/main/java/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -2021,10 +2016,20 @@ public boolean deleteNetwork(long networkId, boolean forced) {
     @Override
     @ActionEvent(eventType = EventTypes.EVENT_NETWORK_RESTART, eventDescription = "restarting network", async = true)
     public boolean restartNetwork(Long networkId, boolean cleanup, boolean makeRedundant, User user) throws ConcurrentOperationException, ResourceUnavailableException, InsufficientCapacityException {
+        NetworkVO network = getNetworkVO(networkId, "Network with specified id doesn't exist", networkId.toString());
+        return restartNetwork(network, cleanup, makeRedundant, user);
+    }
+
+    private NetworkVO getNetworkVO(Long networkId, String errMsgFormat, String objectId) {
```

Comment:
In case objectId is network id always, no need of argument
`String objectId = String.valueOf(networkId)`

```suggestion
    private NetworkVO getNetworkVO(Long networkId, String errMsgFormat) {
```